modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -106,11 +128,11 @@ $ oc delete validatingwebhookconfigurations controller.devfile.io
 If you remove the `devworkspace-webhook-server` deployment without removing the mutating and validating webhooks, you will not be able use `oc exec` commands to run commands in a container on the cluster. After you remove the webhooks you will be able to use the `oc exec` commands again.
```

Comment:
Not part of this PR but would be great if you could add the missing word 'to' here: not be able to use the

Reply:
Good catch! Added.